build-system/tasks/extension-helpers.js

```diff
@@ -488,6 +505,25 @@ async function buildNpmCss(extDir, options) {
   endBuildStep('Wrote CSS', `${options.name} → styles.css`, startCssTime);
 }
 
+/**
+ *
+ * @param {string} extDir
+ * @param {Object} options
+ * @return {Promise<void>}
+ */
+async function buildNpmBentoWebComponentCss(extDir, options) {
+  const srcFilepath = path.resolve(
+    `build/css/${getBentoName(options.name)}-${options.version}.css`
+  );
+  if (await fs.pathExists(srcFilepath)) {
+    const destFilepath = path.resolve(`${extDir}/dist/web-component.css`);
+    await fs.copyFile(srcFilepath, destFilepath);
+  } else {
+    buildExtensionCss(extDir, options.name, options.version, options);
+  }
+  return Promise.resolve();
```

Comment:
can just `return` here, no need to return a promise explicitly since it's an async method